devel/lib/lib.sh

```diff
@@ -23,7 +23,6 @@ export REPO_ROOT="$LIB_ROOT/../.."
 
 export SKIP_BUILD_ADDON_IMAGES="${SKIP_BUILD_ADDON_IMAGES:-}"
 export KIND_CLUSTER_NAME="${KIND_CLUSTER_NAME:-kind}"
-export KIND_IMAGE_REPO="docker.io/kindest/node"
```

Comment:
note: this is removed from this file because `export SERVICE_IP_PREFIX=$(kubectl cluster-info dump...` below causes an ugly error if `devel/lib/lib.sh` is sourced while there's no k8s cluster running (or if kubectl isn't installed :grin: )

latest-kind-images.sh becomes the source of truth for the upstream repo, not this lib.

We might want to change how we do the SERVICE_IP_PREFIX stuff in this file at some point, but I don't think that's a task for this PR